### PR TITLE
all: fix typos

### PR DESCRIPTION
**Pull request by miles-six**
I originally intended to add codespell to the workflow to eliminate all the typos, but I'm concerned that the team may not accept changes to the workflow. 

So, for now, I have only made corrections to the typos. However, if necessary, I can still add the workflow.

The code quality is exceptionally high, so sincerely hope my PR can help make this repository more standardized.

### CODE REVIEW

File: cmd/geth/testdata/vcheck/vulnerabilities.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
...
+    "description": "Affected versions suffer from a vulnerability which can be exploited through the `MULMOD` operation, by specifying a modulo of `0`: `mulmod(a,b,0)`, causing a `panic` in the underlying library. \nThe crash was in the `uint256` library, where a buffer [underflowed](https://github.com/holiman/uint256/blob/4ce82e695c10ddad57215bdbeafb68b8c5df2c30/uint256.go#L442).\n\n\tif `d == 0`, `dLen` remains `0`\n\nand https://github.com/holiman/uint256/blob/4ce82e695c10ddad57215bdbeafb68b8c5df2c30/uint256.go#L451 will try to access index `[-1]`.\n\nThe `uint256` library was first merged in this [commit](https://github.com/ethereum/go-ethereum/commit/cf6674539c589f80031f3371a71c6a80addbe454), on 2020-06-08. \nExploiting this vulnerability would cause all vulnerable nodes to drop off the network. \n\nThe issue was brought to our attention through a [bug report](https://github.com/ethereum/go-ethereum/issues/21367), showing a `panic` occurring on sync from genesis on the Ropsten network.\n \nIt was estimated that the least obvious way to fix this would be to merge the fix into `uint256`, make a new release of that library and then update the geth-dependency.\n",
```
Comment (holiman) [inline cmd/geth/testdata/vcheck/vulnerabilities.json:58]: this file has a corresponding signature, you must not modify this file, so please revert

File: core/state/snapshot/iterator.go [hunk 2]
```diff
@@ -89,7 +89,7 @@ func (dl *diffLayer) AccountIterator(seek common.Hash) AccountIterator {
 	index := sort.Search(len(hashes), func(i int) bool {
 		return bytes.Compare(seek[:], hashes[i][:]) <= 0
 	})
-	// Assemble and returned the already seeked iterator
+	// Assemble and returned the already sought iterator
```
Comment (holiman) [inline core/state/snapshot/iterator.go:92]: pls revert, `seek` is the operation performed

File: core/state/snapshot/iterator.go [hunk 3]
```diff
@@ -255,7 +255,7 @@ func (dl *diffLayer) StorageIterator(account common.Hash, seek common.Hash) (Sto
...
```
Comment (holiman) [inline core/state/snapshot/iterator.go:258]: pls revert, `seek` is the operation performed

File: crypto/secp256k1/libsecp256k1/src/ecmult_impl.h [hunk 4]
```diff
@@ -290,7 +290,7 @@ static void secp256k1_ecmult(const secp256k1_ecmult_context *ctx, secp256k1_gej
 #ifdef USE_ENDOMORPHISM
     secp256k1_ge pre_a_lam[ECMULT_TABLE_SIZE(WINDOW_A)];
     secp256k1_scalar na_1, na_lam;
-    /* Splitted G factors. */
+    /* Split G factors. */
```
Comment (holiman) [inline crypto/secp256k1/libsecp256k1/src/ecmult_impl.h:293]: please revert, this file is imported from upstream. Better to avoid changing it locally

**Comment by miles-six** [comment]
@holiman  already revert them,thanks sir

**Review by lightclient — APPROVED** [review]

**Review by holiman — APPROVED** [review]
LGTM

**Pull request merged**